components/Common/Pagination/index.stories.tsx

```diff
@@ -0,0 +1,99 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+import Pagination from './index';
+
+type Story = StoryObj<typeof Pagination>;
+type Meta = MetaObj<typeof Pagination>;
+
+export const Default: Story = {
+  args: {
+    currentPage: 1,
+    currentPageSiblingsCount: 1,
+    pages: [
+      {
+        url: '1',
+      },
+      {
+        url: '2',
+      },
+      {
+        url: '3',
+      },
+    ],
+  },
+};
+
+export const LeftEllipsis: Story = {
+  args: {
+    ...Default.args,
+    currentPage: 5,
+    pages: [
+      {
```

Comment:
```suggestion
      {  url '1' },
```

All these multiline objects could become one line as they're too small. (Instead of occupying 3 lines per entry, just occupying one)

Reply:
(Suggestion for the whole file)

Reply:
Got it, I will apply those changes